modules/common-workflows/pages/workflow-product-migration-webui.adoc

```diff
@@ -1,33 +1,29 @@
-[[workflow-sp-upgrade-via-webui]]
-= Service Pack Upgrade via {webui}
+[[workflow-product-migration-via-webui]]
+= Product migration via {webui}
 
-
-
-If you want to migrate the registered {sle} client's service pack (SP) to a newer version, it can be done either on the command line or via {webui}.
+If you want to migrate the registered {sle} client pack to a newer product version, it can be done either on the command line or via {webui}.
 
 This document describes and illustrates in detail the migration using the {webui}.
 
 [NOTE]
 ====
-SP versions used are not reflective of the actual latest versions available.
+Product versions used are not reflective of the actual latest versions available.
 They are used for illustration purposes only.
 As an example, the following procedure describes the migration to version 15 SP5 from and older version 15.
 If you are targeting a different version, select the corresponding channels and versions.
 ====
 
-
-.Procedure: Migrating service pack to a newer version using {webui}
+.Procedure: Migrating product to a newer version using {webui}
 [role=procedure]
 
-
-. Log in to {productname} {webui} and navigate to menu:Admin[Products] and search for "SUSE Linux Enterprise Server 15 SP5 x86_64"
+. Log in to {productname} {webui} and navigate to menu:Admin[Products] and search for [literal]``SUSE Linux Enterprise Server 15 SP5 x86_64``.
 . Select the recommended channels.
 +
 image::upgradesles01.png[scaledwidth=80%]
 . Click btn:[Add Products].
 +
 image::upgradesles02.png[scaledwidth=80%]
-. Navigate to menu:Systems[registered client > Software > SP Migration].
+. Navigate to menu:Systems[Registered client > Software > SP Migration].
```

Comment:
```suggestion
. Navigate to menu:Systems[Registered client > Software > Product Migration].
```

Reply:
TBD at a sync meeting: Do we really want to call this `Systems[Registered client]`?  I think initially we called this the `System Details` page (or tab).

Reply:
Added to the Confluence.